core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -3152,9 +3157,12 @@ private void o3MoveUncommitted0(
                 long sourceAddress = srcDataMem.addressOf(srcFixOffset);
                 Vect.memcpy(appendAddress, sourceAddress, extendedSize);
             } else {
-                long sourceAddress = mapRO(ff, srcDataMem.getFd(), extendedSize, srcFixOffset, MemoryTag.MMAP_TABLE_WRITER);
-                Vect.memcpy(appendAddress, sourceAddress, extendedSize);
-                ff.munmap(sourceAddress, extendedSize, MemoryTag.MMAP_TABLE_WRITER);
+                // Linux requires the mmap offset to be page aligned
+                long alignedOffset = ff.alignedOffset(srcFixOffset);
+                long alignedExtraSize = srcFixOffset - alignedOffset;
```

Comment:
same name as first time would be better `alignedExtraLen`

Reply:
Renamed in 9678f2dec1966a917f63dd78de06e01c1d2d0740